### PR TITLE
updates to address recursive decisions

### PR DESCRIPTION
**Pull request by laurie-tyz**
this is where I get stuck!

### CODE REVIEW

**Comment by ahouseholder** [comment]
Fixes #74

File: doc/md_src_files/040_stakeholders-scope.md [hunk 1]
```diff
@@ -114,9 +114,14 @@ Table 2: Proposed Meaning for Supplier Priority Outcomes
...
+
+Mitigation techniques may not affect the SSVC decision. For example, the implementation of a firewall or IDS rule to mitigate [*System Exposure*](#system-exposure) from open to controlled is only valid until someone changes the rule. In the area of Financial impacts, the caps on the insurance may be too low to act as a mitigation. The Physical impact may be increased by incorrect installation of the physical barriers designed to restrict a robot’s ability to interact with humans. The Mission impact could be increased when a disaster recovery test-run identifies problems with an alternate business flow.
```
Comment (j---) [inline doc/md_src_files/040_stakeholders-scope.md:122]: After

> Mitigation techniques may not affect the SSVC decision.

add:
The mitigating action may not be permanent or work as designed.

File: doc/md_src_files/040_stakeholders-scope.md [hunk 2]
```diff
@@ -114,9 +114,14 @@ Table 2: Proposed Meaning for Supplier Priority Outcomes
...
+
+A mitigation that successfully changes the value of a decision point may shift the priority of further action to a reduced state. Implementing an adequate change control process for maintaining firewall or IDS rules may be enough to reduce the priority where no further action is necessary. In the area of Financial impacts, a better insurance policy may be purchased, providing necessary fraud insurance. Physicial well-being impact may be reduced by testing the physicial barriers designed to restrict a robot's ability to interact with humans. Mission impact could be reduced by correcting the problems identified in a disaster recover test-run of the alternate business flow. If applying a mitigation reduces the priority to **defer**, the deployer may not need to apply a remediation, later if it becomes available. Table 3 displays the action priorities for the deployer, which are similar to the supplier case.
```
Comment (j---) [inline doc/md_src_files/040_stakeholders-scope.md:124]: s /  later if it becomes / if it later becomes /

Comment (j---) [inline doc/md_src_files/040_stakeholders-scope.md:124]: > Implementing an adequate change control process for maintaining firewall or IDS rules 

Suggest instead:
An effective firewall or IDS rule coupled with an adequate change control process for rules

**Review by j--- — CHANGES_REQUESTED** [review]
Besides these 3 minor changes, looks good.

File: doc/md_src_files/040_stakeholders-scope.md (hunk 1 above)
Comment (ahouseholder) [reply doc/md_src_files/040_stakeholders-scope.md]: done

File: doc/md_src_files/040_stakeholders-scope.md (hunk 2 above)
Comment (ahouseholder) [reply doc/md_src_files/040_stakeholders-scope.md]: 81d2fd6

**Review by ahouseholder — APPROVED** [review]
I made a few quick edits based on Jono's review. Looks good to me as of latest commit.

**Review by j--- — APPROVED** [review]

**Review by cgyarbrough — APPROVED** [review]
These changes look like what we discussed. Approved.

**Pull request merged**